client/blocks/comment-detail/comment-detail-actions.jsx

```diff
@@ -22,12 +23,13 @@ const commentActions = {
 const hasAction = ( status, action ) => includes( commentActions[ status ], action );
 
 export const CommentDetailActions = ( {
+	siteId,
+	postId,
+	commentId,
```

Comment:
I always alphabetize props, destructuring, etc.
On the other hand, I find clearer to have IDs on top (also ordered after the actions/selectors arguments), so yeah, another 100% personal preference. :)